modules/avm/variables.tf

```diff
@@ -1,3 +1,21 @@
+variable "datadog_api_key" {
+  type        = string
+  default     = null
+  description = "Datadog API key"
+}
+
+variable "datadog_install_log_forwarder" {
+  type        = bool
+  default     = false
+  description = "Set to true to install Datadog AWS Log Forwarder"
+}
+
+variable "datadog_integration" {
+  type        = bool
+  default     = false
+  description = "Whether the Datadog integration should be enabled or not"
+}
```

Comment:
Could we use a single `datadog` variable here?

Reply:
Do you mean a single variable to control both the integration and the log forwarding?